packages/data-stores/src/plans/queries/use-plans.ts

```diff
@@ -12,7 +12,7 @@ interface PlansIndex {
 /**
  * Plans from `/plans` endpoint, transformed into a map of planSlug => PlanNext
  */
-function usePlans( { coupon }: { coupon: string | undefined } ): UseQueryResult< PlansIndex > {
+function usePlans( { coupon }: { coupon?: string } = {} ): UseQueryResult< PlansIndex > {
```

Comment:
This was an intentional change, to not have this defined as optional. It's far more risky to easily allow this  hook to fetch pricing data without a coupon code when one is needed. I want to make sure devs are opinionated/concious about passing undefined here.

Do you mind reverting this change please and following up with passing `{ coupon: undefined }`? (or maybe @jdc91 since you've approved?)

Reply:
@chriskmnds I don't think that changing an interface retroactively, risking fatals across the whole platform, is a good strategy to ensure developers follow a new requirement they don't know about.

Reply:
This is a new interface and i'm still working out things around it. It's my bad that it caused fatals, but i'd appreciate if this change is reverted. It's been a very long day.